src/Our.Umbraco.RobotsTxtEditor/Web/UI/App_Plugins/RobotsTxtEditor/js/robotstxteditorController.js

```diff
@@ -47,11 +48,12 @@ angular.module("umbraco")
                         // TODO: See if there is a nice way of doing this whilst maintaining correct format
```

Comment:
I'll have a think about this. In the legacy version, I used an embedded-resource to populate the default/initial robots.txt contents.  This isn't a showstopper. 🙂